_docs/software_support_lifecycle.md

```diff
@@ -0,0 +1,80 @@
+---
+
+## Software Support Lifecycle
+
+As a part of Vox Pupuli's mission to provide a home for modules that still have a userbase and need development, we need to define a clear support strategy for what operating systems, dependencies, and upstream software we will support and for how long those will be supported.
+
+As we operate entirely on the graces of our community members being willing to voluntarily contribute to our modules, we also need to be respective of their time and resources and not add undue stress in order to support software that upstream creators no longer support.
+
+To account for this, we no longer support any software or operating system after the date that the maintainers have marked said software or operating system as End of Life. However, this does not mean we will immediately remove code that allows for the module to run on End of Life'd software or operating systems, but the community will no longer be expected to provide support to users for that software.
+
+There are exceptions and the full policy is detailed in the next section.
+
+### Support Lifecycle
+
+#### Operating Systems
+An Operating System is no longer considered supported once it reaches an End of Life date determined by the upstream maintainer* of the operating system. To ensure our users have enough time to adjust, we will take the following steps to ensure as seamless of a transition as possible:
+
+1. A deprecation PR will be created no later that 3 months before an Operating System's End of Life date.
+2. Removal of the End of Life version from the metadata.json and tests to be completed by the End of Life date.
+3. Issues relating to the End of Life operating system will no longer be addressed after the End of Life date.
+4. Code allowing a module to be run on the End of Life operating system will be left in the module until one of the following milestones are reached after which time the code will be removed:
+    1. Maintaining the code prevents the module from moving forward.
+    2. 12 Months after the End of Life date.
+5. Pull Requests will continue to be accepted with the following exceptions:
+    1. Accepting such a pull request would be detrimental to the module's long term goals.
+    2. The code has already been removed as per the standards set in section 4.
+
+*maintainers include, but are not limited to, the community, company, or organization that maintains ownership over a specific piece of software or project.
+
+##### Support Table:
+
+| OS Name | Support Begins | End of Life | Notification PR | End of Support | Removal of Code |
+| ------- | -------------- | ----------- | --------------- | -------------- | --------------- |
+| CentOS/RHEL 6 | July 10th, 2011 | Nov 30th, 2020 | N/A | Nov 30th, 2020 | Nov 30th, 2021 |
+| CentOS/RHEL 7 | July 7th, 2014 | June 30th, 2024 | Mar 31st, 2024 | June 30th, 2024 | June 30th, 2025 |
+| CentOS/RHEL 8 | Sept 24th, 2019 | Dec 31, 2021 | Sept 30th, 2021 | Dec 31, 2021 | Dec 31, 2022 |
```

Comment:
RHEL 8 of course won't be EOL next year.

Reply:
I think it's a 'wait and see' for now as to how we test RHEL 8 when CentOS 8 disappears.  Maybe we just have to use CentOS 8.3 images even when RHEL is on eg 8.8???

Reply:
According to CentOS' FAQ, we need to be testing for RHEL on RHEL images going forward :shrug: 

@ekohl mentioned that there seems to be some confusion around this though.

Reply:
For those unaware, I have sent an email about this with the PMC on CC. There is nothing secret in [the initial email](https://gist.github.com/ekohl/6418d1435b9f12a394256bb0367c6a0b). To set the expectations right, I think the people responsible for that alias are currently busy but they are working on a solution exactly for the problem we're having: CI to test integration for community projects. While I work for Red Hat, this is not an area I work on and can only advocate for it. Luckily, our use case is not special in any way (loads of projects can/should do CI like we do) and one that was explicitly mentioned to be a focus point.

Reply:
A Red Hat Developer subscription for the Vox Pupuli organization would almost work, if not for the "single-user" limitations. https://developers.redhat.com/articles/faqs-no-cost-red-hat-enterprise-linux

Reply:
I have been told they want to fix this. Since it's a license, legal is involved. That means we need to be patient. The holiday season probably doesn't help either.